cfme/services/catalogs/orchestration_template.py

```diff
@@ -1,166 +1,284 @@
+
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.is_displayed and
+            self.title.text == "Copying {}".format(self.obj.name) and
+            self.orchestration_templates.is_opened
+        )
+
+
+class DetailsTemplateView(TemplateForm):
+
+    @property
+    def is_displayed(self):
+        return (
+            self.title.text == '{} "{}"'.format(self.obj.template_type[:-1],
+                                                self.obj.template_name) and
+            self.orchestration_templates.is_opened
+        )
+
+
+class TemplateTypeView(TemplateForm):
+
+    @property
+    def is_displayed(self):
+        return (
+            self.title.text == '{} "{}"'.format(self.obj.template_type) and
+            self.orchestration_templates.is_opened
+        )
+
+
+class DialogForm(ServicesCatalogView):
+    title = Text('#explorer_title_text')
+
+    name = Input(name='dialog_name')
+
+
+class AddDialogView(DialogForm):
+
+    add_button = Button("Save")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.title.text == 'Adding a new Service Dialog from '
+                               'Orchestration Template "{}"'.format(self.obj.name) and
+            self.orchestration_templates.is_opened
+        )
+
+
+class OrchestrationTemplate(Updateable, Pretty, Navigatable):
+
+    def __init__(self, template_type=None, template_name=None, description=None,
+                 appliance=None):
+        Navigatable.__init__(self, appliance)
         self.template_type = template_type
         self.template_name = template_name
         self.description = description
 
-    def create_form(self, content):
-        sel.force_navigate('create_new_template',
-                           context={'template_type': self.template_type})
+    def create(self, content):
+        view = navigate_to(self, "AddTemplate")
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
         else:
             temp_type = "OpenStack Heat"
-        fill(create_template_form, {'description': self.description,
-                                    'template_type': temp_type,
-                                    'content': content,
-                                    'template_name': self.template_name},
-            action=create_template_form.add_button)
-
-    def create(self, content):
-        self.create_form(content)
-        try:
-            flash.assert_success_message('Orchestration Template "{}" was saved'.format(
-                self.template_name))
-        except Exception as e:
-            if error.match("Error during 'Orchestration Template creation': Validation failed: \
-                            Md5 of content already exists (content must be unique)", e):
-                self.create_form(content + " ")
-            raise
+        view.fill({'name': self.template_name,
```

Comment:
@psav unrelated to this pr, we do seem to have a lot of alinged indents like that, and i feel like things would look more beautiful if we just introduced a newline and normal followup indent